packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -114,8 +120,10 @@ export class VerticalStackedBarChartBase extends React.Component<
     this._handleMouseOut = this._handleMouseOut.bind(this);
     this._calloutId = getId('callout');
     this._tooltipId = getId('VSBCTooltipId_');
-    this._adjustProps();
-    this._dataset = this._createDataSetLayer();
+    if (this.props.data && this.props.data.length > 0) {
```

Comment:
Dont add check to this code. The adjustProps call is setting some prop values

Reply:
looks like we need to add this check as internally it refers to "this.props.data[0].xAxisPoint" which crashes otherwise as follows:
<img width="649" alt="image" src="https://user-images.githubusercontent.com/120183316/231732784-cf8813d3-ba2f-48a8-9691-918682b65ac2.png">

Reply:
Did you remove this check. I dont see it in the code anymore